ProjectLighthouse/Helpers/VersionHelper.cs

```diff
@@ -12,64 +11,28 @@ static VersionHelper()
     {
         try
         {
-            CommitHash = ResourceHelper.ReadManifestFile("gitVersion.txt");
-            Branch = ResourceHelper.ReadManifestFile("gitBranch.txt");
+            CommitHash = ThisAssembly.Git.Commit;
+            Branch = ThisAssembly.Git.Branch;
             string commitNumber = $"{CommitHash}_{Build}";
             FullRevision = Branch == "main" ? $"r{commitNumber}" : $"{Branch}_r{commitNumber}";
```

Comment:
Is the try/catch still necessary here seeing as these values are constants?

Reply:
> Is the try/catch still necessary here seeing as these values are constants?

yeah it seems the try/catch was entirely there to handle errors caused by missing/malformed txt files

Reply:
Removed the try catch and the entire constructor since it was unnecessary